cmd/consensus/main.go

```diff
@@ -695,12 +668,45 @@ func loadDKGPrivateData(dir string, myID flow.Identifier) (*dkg.DKGParticipantPr
 	return &priv, nil
 }
 
-func loadMachineAccountPrivateData(node *cmd.FlowNodeBuilder) (*bootstrap.NodeMachineAccountInfo, error) {
-	data, err := io.ReadFile(filepath.Join(node.BaseConfig.BootstrapDir, fmt.Sprintf(bootstrap.PathNodeMachineAccountInfoPriv, node.Me.NodeID())))
+func createDKGContractClient(node *cmd.FlowNodeBuilder, accessAddress string) (module.DKGContractClient, error) {
```

Comment:
This PR https://github.com/onflow/flow-go/pull/903 implements the issue that was marked as a blocker for this. The main change there is that well-known system contract addresses are embedded in a package rather than passed in as a flag. Really the blocker is just that there will be conflicts between the two PRs, where https://github.com/onflow/flow-go/pull/903 removes the `--qc-contract-address` flag and this PR makes use of it.

Reply:
Yeah that makes sense. I'll resolve the conflicts once #903 is merged.

Reply:
FYI it's merged now

Reply:
Fixed